tests/test_pihong_dossier_1490.py

```
@@ -3374,3 +3407,663 @@ def test_657_resume_phase2_signal_empty_desk_http(web_game, monkeypatch):
+    session = GameSession.__new__(GameSession)
+    session.db = db
+    session.state = state
+    session.llm_config = None
+    session.content = content
+    session._refuse_if_settling = lambda: None  # type: ignore[attr-defined]
+
+    # 自由下旨真入口：坏 shape 不得落 directive / 御笔
+    for bad in (True, 1.9, "1"):
+        with pytest.raises(ValueError):
+            session.add_directive(
+                f"着即强推（坏shape）",
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)